pkg/vm/vm_test.go

```diff
@@ -197,6 +197,30 @@ func TestPushData4Good(t *testing.T) {
 	assert.Equal(t, []byte{1, 2, 3}, vm.estack.Pop().Bytes())
 }
 
+func callNTimes(n uint16) []byte {
+	buf := make([]byte, 2)
+	binary.LittleEndian.PutUint16(buf, n)
+
+	return makeProgram(
+		PUSHBYTES2, Instruction(buf[0]), Instruction(buf[1]),
```

Comment:
Is it worth making a buf and going through `binary`? `Instruction(byte(n&0xff)), Instruction(byte((n>>8)&0xff) // little endian n.`

Reply:
Or even `Instruction(byte(n)), Instruction(byte((n>>8)) // Little endian n`, it's Go.